app/models/author_book.rb

```diff
@@ -1,4 +1,8 @@
 class AuthorBook < ApplicationRecord
-belongs_to :author
-belongs_to :book
-end
\ No newline at end of file
+ belongs_to :author
+ belongs_to :book
+
```

Comment:
Should we have this?

Reply:
Yes, after going through the  lesson this morning. I can confirm we do need the code.  The lesson also does have the following: 

class PlaylistSong < ApplicationRecord
  belongs_to :playlist
  belongs_to :song
end